graphql/pupil/mutations.ts

```diff
@@ -247,7 +267,7 @@ async function pupilRegisterPlus(data: PupilRegisterPlusInput, ctx: GraphQLConte
 @Resolver((of) => GraphQLModel.Pupil)
 export class MutatePupilResolver {
     @Mutation((returns) => Boolean)
-    @Authorized(Role.PUPIL, Role.ADMIN, Role.PUPIL_SCREENER)
+    @Authorized(Role.ADMIN, Role.PUPIL_SCREENER)
```

Comment:
Um, this mutation is (was?) used on the settings page by pupils to edit their own account? ... Was that removed?